lib/endpoints/volumes.js

```diff
@@ -442,7 +526,7 @@ function validateUpdateVolumeInput(req, res, next) {
         validationErrs.push(latestErr);
     }
 
-    for (paramName in req.params) {
+    for (paramName in req.query) {
```

Comment:
>  But I believe req.params.id will be validated by the call to

Ah right, somehow I had missed that `volumesValidation.validateVolumeId(req.params.id);` call. Thanks!